charts/sysdig/values.yaml

```diff
@@ -247,6 +247,117 @@ nodeImageAnalyzer:
       cpu: 500m
       memory: 1024Mi
 
+nodeAnalyzer:
+  deploy: false
+
+  # The API endpoint for Sysdig Secure, specified with no protocol:
+  # * SaaS default region (US East): leave empty
+  # * SaaS US Web: us2.app.sysdig.com
+  # * SaaS European Union: eu1.app.sysdig.com
+  # * On-Prem: sysdig.my.company.com
+  # collectorEndpoint: sysdig.my.company.com
+
+  # Can be set to false to allow insecure connections to the Sysdig backend,
+  # such as for on-premise installs that use self-signed certificates.
+  # By default, certificates are always verified.
+  # sslVerifyCertificate: false
+
+  # Can be set to true to show debug logging, useful for troubleshooting.
+  debug: false
+
+  # Proxy configuration variables. See also: [Running Node Image Analyzer Behind a Proxy](https://docs.sysdig.com/en/scan-running-images.html#UUID-b3b07aa6-db02-eb58-050f-15c9e053bb64_section-idm232105909710949)
+  httpProxy:
+  httpsProxy:
+  noProxy:
+
+  # Allow sysdig Node Image Analyzer to run on Kubernetes 1.6 masters
+  tolerations:
+    - effect: NoSchedule
+      key: node-role.kubernetes.io/master
+
+  # Use this pullSecret to pull images from a private registry
+  # pullSecrets:
+  #   - name: myRegistryKeySecretName
+
+  imageAnalyzer:
+    image:
+      repository: sysdig/node-image-analyzer
+      tag: 0.1.12
+      pullPolicy: IfNotPresent
+
+    settings:
+    # The Docker socket path.
+    # If a custom path is specified, ensure it is correctly mounted from the host inside the container.
+    # dockerSocketPath: unix:///var/run/docker.sock
+
+    # The socket path to a CRI compatible runtime, such as CRI-O.
+    # If a custom path is specified, ensure it is correctly mounted from the host inside the container.
+    # criSocketPath: unix:///var/run/crio/crio.sock
+
+    # The socket path to a CRI-Containerd daemon.
+    # If a custom path is specified, ensure it is correctly mounted from the host inside the container.
+    # containerdSocketPath: unix:///var/run/containerd/containerd.sock
+
+    # Allow passing extra volumes to the Node Image Analyzer to mount docker socket, cri-o socket, etc.
+    extraVolumes:
+      volumes: []
+      mounts: []
+
+      # Example:
+
+      # volumes:
+      # - name: docker-sock
+      #   hostPath:
+      #     path: /var/run/docker.sock
+      # mounts:
+      # - mountPath: /var/run/docker.sock
+      #   name: docker-sock
+    resources:
+      requests:
+        cpu: 150m
+        memory: 512Mi
+      limits:
+        cpu: 500m
+        memory: 1536Mi
+  hostAnalyzer:
+    image:
+      repository: sysdig/host-analyzer
+      tag: 0.1.10 # TODO pinned or latest?
```

Comment:
What should this tag be?

Reply:
We use pinned versions following the convention: 

https://helm.sh/docs/chart_best_practices/pods/#images

Although it involves a bit of more work in our side, I would keep doing it.

Reply:
Agreed =) I just need the latest version so I can use that

Reply:
The CI failed, I think is because the comment

Once you remove it, it will be fine